pkg/apis/garden/v1beta1/types.go

```diff
@@ -1166,6 +1166,9 @@ type KubernetesDashboard struct {
 // ClusterAutoscaler describes configuration values for the cluster-autoscaler addon.
 type ClusterAutoscaler struct {
 	Addon `json:",inline"`
+	// ScaleDownUtilizationThreshold defines the threshold in % under which a node is being removed
+	// +optional
+	ScaleDownUtilizationThreshold *string `json:"scale-down-utilization-threshold,omitempty"`
```

Comment:
```suggestion
	ScaleDownUtilizationThreshold *string `json:"scaleDownUtilizationThreshold,omitempty"`
```

Reply:
This should follow K8S naming conventions - see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions

Reply:
Good point!